app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] [OPTIONAL] Kindly consider adding `foreign_keys` to the has_many relationships 

```suggestion
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
- Well resolved 👍🏽